src/posts/lucky_0.29.cr

```diff
@@ -0,0 +1,192 @@
+class Lucky029Release < BasePost
+  title "Happy Holidays and welcome Lucky 0.29"
+  slug "lucky-0_29-release"
+
+  def published_at : Time
+    Time.utc(year: 2021, month: 12, day: 8)
+  end
+
+  def summary : String
+    <<-TEXT
+    The Lucky 0.29 release brings us many more
+    quality of life updates, bug fixes, and a
+    few new features to really make your app
+    solid.
```

Comment:
Not that it matters, but why did you wrap to 43 characters?

Reply:
because my font is super zoomed in, and that's the edge 😂

Reply:
oh wait... this is on the upper part... umm no clue why I did this lol